### PR TITLE
Replaced "the" article with "a". Replaced the past tense with the present tense. 

### PR DESCRIPTION
**Pull request by veryniceusername**
Replaced "the" article with "a".
Replaced the past tense with the present tense.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/http/headers/last-modified/index.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ browser-compat: http.headers.Last-Modified
 ---
 {{HTTPSidebar}}
 
-The **`Last-Modified`** response HTTP header contains the date
-and time at which the origin server believes the resource was last modified. It is used
-as a validator to determine if a resource received or stored is the same. Less accurate
+The **`Last-Modified`** response HTTP header contains a date
+and time when the origin server believes the resource was last modified. It is used
+as a validator to determine if a resource receives or stores the same. Less accurate
```
Comment (teoli2003) [inline files/en-us/web/http/headers/last-modified/index.md:15]: This seems to change a bit the meaning. What about
```suggestion
as a validator to determine if the resource is the same as the previously stored one. Less accurate
```

(Feel free to iterate here)

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One change seems to modify the meaning a little bit; overall it is a good improvement!

File: files/en-us/web/http/headers/last-modified/index.md (hunk 1 above)
Comment (annakhyzhnya) [reply files/en-us/web/http/headers/last-modified/index.md:15]: yeap, I did not get the meaning. This one "as a validator to determine if the resource is the same as the previously stored one" is correct. Thanks)

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks for the PR and your time yesterday! All fine!

**Pull request merged**